docs/integration.md

```diff
@@ -0,0 +1,102 @@
+# Integrate with Rafiki
+
+**❗ Rafiki is intended to be run by [Account Servicing Entities](./glossary.md#account-servicing-entity) only and should not be used in production by non-regulated entities.**
+
+Account Servicing Entities provide and maintain payment accounts. In order to make these accounts Interledger-enabled via Rafiki, they need to provide the following endpoints and services.
+
+## Quotes / Fees
+
+Every Interledger payment is preceded with a quote that estimates the costs for transfering value from A to B. The Account Servicing Entity may charge fees on top of that for facilitating that transfer. How they structure those fees is completely up to the Account Servicing Entity.
+
+The Account Servicing Entity is required to provide an endpoint that is accessible to the Rafiki backend. It accepts a `POST` request with
+
+#### Request Body
+
+| Variable Name    | Type                                     | Description                                 |
+| ---------------- | ---------------------------------------- | ------------------------------------------- |
+| id               | string                                   | Interledger quote id                        |
+| paymentType      | enum: `'FixedSend'` \| `'FixedDelivery'` | fixed-send or fixed-receive payment         |
+| paymentPointerId | string                                   | id of sending payment pointer               |
+| receiver         | string                                   | receiving payment pointer                   |
+| sendAmount       | [Amount](#amount)                        | defined or quoted send amount               |
+| receiveAmount    | [Amount](#amount)                        | defined or quoted receive amount            |
+| createdAt        | string                                   | creation date and time of Interledger quote |
+| expiresAt        | string                                   | expiry date and time of Interledger quote   |
+
+#### Amount
+
+(The example amount is $42.42.)
+
+| Variable Name | Type             | Description                                                                                               |
+| ------------- | ---------------- | --------------------------------------------------------------------------------------------------------- |
+| value         | number // bigint | e.g. `4242`                                                                                               |
```

Comment:
`number // bigint`

it's not one or the other ?

Reply:
This is up to the Account Servicing Entity. They can actually provide us with `string`, `number`, or `bigint`. I'll add the string type.